lib/screens/downloads_settings_screen.dart

```diff
@@ -1,26 +1,325 @@
+import 'dart:io';
+import 'dart:math';
+
+import 'package:finamp/components/AlbumScreen/download_button.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:get_it/get_it.dart';
+import 'package:hive/hive.dart';
 
-import 'add_download_location_screen.dart';
-import '../components/DownloadLocationSettingsScreen/download_location_list.dart';
+import '../components/global_snackbar.dart';
+import '../models/finamp_models.dart';
+import '../services/finamp_settings_helper.dart';
+import '../services/isar_downloads.dart';
+import 'downloads_location_screen.dart';
 
 class DownloadsSettingsScreen extends StatelessWidget {
-  const DownloadsSettingsScreen({Key? key}) : super(key: key);
+  const DownloadsSettingsScreen({super.key});
 
-  static const routeName = "/settings/downloadlocations";
+  static const routeName = "/settings/downloads";
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text(AppLocalizations.of(context)!.downloadLocations),
+        title: Text(AppLocalizations.of(context)!.downloadSettings),
       ),
-      floatingActionButton: FloatingActionButton(
-        child: const Icon(Icons.add),
-        onPressed: () => Navigator.of(context)
-            .pushNamed(AddDownloadLocationScreen.routeName),
+      body: ListView(
+        children: [
+          ListTile(
+            leading: const Icon(Icons.folder),
+            title: Text(AppLocalizations.of(context)!.downloadLocations),
+            onTap: () => Navigator.of(context)
+                .pushNamed(DownloadsLocationScreen.routeName),
+          ),
+          const RequireWifiSwitch(),
```

Comment:
This should only be shown on Android and iOS: https://github.com/jmshrv/finamp/pull/568#issuecomment-1935467627